src/lib/addons/datadog-definition.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const {
+    FEATURE_CREATED,
+    FEATURE_UPDATED,
+    FEATURE_ARCHIVED,
+    FEATURE_REVIVED,
+    FEATURE_STALE_ON,
+    FEATURE_STALE_OFF,
+} = require('../types/events');
+
+module.exports = {
+    name: 'datadog',
+    displayName: 'Data dog',
+    description: 'Allows Unleash to post updates to Data dog.',
+    documentationUrl: 'https://docs.getunleash.io/docs/addons/datadog',
+    parameters: [
+        {
+            name: 'url',
+            displayName: 'Data dog webhook URL',
+            type: 'url',
+            required: true,
+            sensitive: true,
```

Comment:
This field does not have to be sensitive if it does not contain any keys in the url itself.

Reply:
can we have a default value for this?